README.md

```diff
@@ -72,17 +75,3 @@ $ yarn add @fortawesome/fontawesome-svg-core \
 
 See the `package.json` file for each package to see what dependencies are required.
 
-## Storybook
-
-There is a global storybook, which showcases the components from all packages.
-
-https://stenait.github.io/stenajs-webui
-
-You can start it locally:
```

Comment:
I still think we want the frist section of StoryBook to be in the readme. It is only the part where we tell the user how to start it locally that should be removed.

Reply:
Create a task!

Reply:
Oh, wait, I misread.

I added a link in the top of README to Storybook.